extras/gomock/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+alias(
+    name = "gomock",
+    actual = "@com_github_golang_mock//gomock",
```

Comment:
What's the purpose of introducing these aliases?

Reply:
With bzlmod, modules can only see repositories they explicitly depend on in their `MODULE.bazel`. Without there's aliases, rules_go users would need to depend on Gazelle themselves and import the `@com_github_golang_gomock` repo via a `use_repo` directive just to use the rules_go provided `gomock` macro.

That seemed very tedious, so I introduced these aliases instead.